src/OSPSuite.Core/Domain/SensitivityAnalyses/OutputParameterSensitivity.cs

```diff
@@ -17,6 +17,12 @@ public class OutputParameterSensitivity
       /// </summary>
       public float[] OutputValues { get; }
 
+      /// <summary>
+      ///    Simulated time for the given <see cref="OutputPath" />
+      /// </summary>
+      public float[] TimeValues { get; }
```

Comment:
All right  I have added a TimeValues. At the moment, each output is going to get a reference to this array. Becaus each results is seen as a different individuals, we will have one for each variation. Probably something to optimize since they should normally all be the same. But good enough for now